### PR TITLE
Custom rpc + new texts

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: multisig-watcher/src/agent-ethereum.ts [hunk 1]
```diff
@@ -7,10 +7,18 @@ import {
   FindingSeverity,
 } from "forta-agent";
 
-import { GNOSIS_SAFE_EVENTS_OF_NOTICE, SAFES_ETH } from "./constants";
+import {
+  Blockchain,
+  GNOSIS_SAFE_EVENTS_OF_NOTICE,
+  SAFES,
```
Comment (vgorkavenko) [inline multisig-watcher/src/agent-ethereum.ts:13]: Use `SAFES` or `safesGlobal` in all cases. I propose using `SAFES` everywere

File: multisig-watcher/src/config/bot-config.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
 {
     "Arbitrum": {
-        "RpcUrl": "https://arb1.arbitrum.io/rpc"
+        "RpcUrl": "aHR0cHM6Ly9hcmItbWFpbm5ldC5nLmFsY2hlbXkuY29tL3YyL04zR3M2VnRQU2hUZmFHOUV2UjhMU2g0RUVHYWdMdW5x"
```
Comment (vgorkavenko) [inline multisig-watcher/src/config/bot-config.json:3]: I think it should be renamed to `RpcUrlBase64`, for example

File: multisig-watcher/src/agent-ethereum.ts (hunk 1 above)
Comment (dgusakov) [reply multisig-watcher/src/agent-ethereum.ts:13]: Fixed

File: multisig-watcher/src/config/bot-config.json (hunk 2 above)
Comment (dgusakov) [reply multisig-watcher/src/config/bot-config.json]: Updated

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**